frontend/src/css/style.css

```diff
@@ -671,3 +671,8 @@ button:disabled {
     }
 
 }
+
+.arrow-back {
```

Comment:
Why do you use a class if you use a svg. You have property in svg tag to set height and width ?

Reply:
because in his defaut css his width and heigth is already set and take the ascendancy on property